sites/docs/src/content/docs/tutorials/switching_barnch_to_main/switching_master_to_main.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Transitioning from `master` to `main` branches in pipelines"
+subtitle: Guide on how to switch a pipeline from the default branch `master` to `main`
+weight: 20
+---
+
+This tutorial will guide you through changing the pipeline default branch form `master` to `main`.
+
+## Prequisites
+
+- Admin rights on the GitHub repository where the change taking place
+- nf-core tools
+  - It is recommended that the version match the version of the template in your pipeline
+
+> [!WARNING]
+> Make sure you are on using same version of nf-core/tools as the template version of your branch.
+> This will avoid adding other changes not needed for the branch name switch.
+
+## Instructions
+
+1. Open your repository on GitHub (make sure you're on master)
+2. Go to the `branches` page (to the right of the `branches` drop down where it says 'master')
+
+![image](https://hackmd.io/_uploads/rkc8GXJWgx.png)
+
+3. Under the default branch section of the page, press the triple dot menu on the `master` branch row, and press 'Rename branch'
+
+> [NOTE]
+> You will need permissions to change this setting. If you don't have permission, ask the @core-team to do it for you.
+
+![image](https://hackmd.io/_uploads/Bkpdz7kWlg.png)
+
+4.  Rename `master` to `main`
+5.  Press the 'learn more' text, and copy and paste the displayed instructions somewhere safe.
+
+> [NOTE]
+> These instructions will be usefur for all pipeline contributors.
+
+![image](https://hackmd.io/_uploads/SkiIQXyWlx.png)
+
+6. Press 'Rename branch'
+7. Go back to 'Code' tab to verify you are now on `main`
+8. In your local IDE (e.g. VSCode) make sure you're on the `dev` branch
+9. Run `git pull` to ensure you have the `main` branch locally
+10. Check your `git` config to see what is your current default branch (it should report `master`):
+
+    ```bash
+    git config --global init.defaultBranch
+    ```
+
+11. Change it to `main` with :
+
+    ```bash
+    git config --global init.defaultBranch main
+    ```
+
+12. Check your `git` config again to check it change (it should now report `main`):
+
+    ```bash
+    git config --global init.defaultBranch
+    ```
+
+13. Still on `dev`, run `nf-core pipelines sync`
+14. Change to a new branch `git switch -c default-branch-change`
+15. Follow the merge TEMPLATE instructions, i.e. `git merge TEMPLATE`
```

Comment:
```suggestion
15. Follow the [merge TEMPLATE instructions](https://nf-co.re/docs/tutorials/sync/merging_automated_prs#resolving-merge-conflicts)
```

Reply:
I think these docs are way too complicated for this purpose, there will be few changes